app/models/concerns/formatable.rb

```diff
@@ -0,0 +1,27 @@
+module Formatable
+  def format_city_state_zip(data)
+    city = data[:city]
+    state_abbr = data[:state]
+    zip_code = data[:zip_code]
+
+    "#{city}, #{state_abbr} #{zip_code}"
+  end
+
+  # TODO: Update method to handle different timezones
+  # Returns the date in the format of 'Wednesday, Sep 22, 2021 9:41pm'
+  def format_date
+    DateTime.strptime(date_time, '%Y-%m-%dT%H:%M:%S')
+            .strftime('%A, %b %d, %Y%l:%M%P')
+  end
+
+  # TODO: Update method to handle different timezones
+  # Returns the date in the format of 'Wed 09/22/21- 9:41pm'
+  def format_date_short
+    DateTime.strptime(date_time, '%Y-%m-%dT%H:%M:%S')
+            .strftime('%a %m/%d/%y-%l:%M%P')
+  end
+
+  def date
+    DateTime.strptime(date_time, '%Y-%m-%dT%H:%M:%S').strftime('%a %m/%d/%y')
+  end
```

Comment:
@tvaroglu To make these methods more flexible, I think we could provide the `date_time` as an argument to the methods.

Also, if we include these as view helper methods for the views (within `app/helpers` as either `date_helper.rb` or `format_helper.rb`), we can call something like `format_date_short(event.date_time)` in the view `.html.erb` files and it would render the formatted date.  Sorry for misleading you with the `formatable`!

I think using a view helper makes more sense for this because we are only formatting for how the date is displayed in the view.  If the methods were used to do some other manipulation of the business logic, then a model module would make more sense.

Check out this article on helpers: [LINK](https://www.rubyguides.com/2020/01/rails-helpers/)

Reply:
Shoot... missed this and just caught your comment post-merge.  Just added an issue to the FE board to track resolution.  Devil's advocate, for now, it probably makes sense to keep continuity for things that 'format' within the 'formatable` module, and if these methods *do* end up supporting business logic, then it makes even more sense to keep in one place (IMO).